assets/scss/common/_custom.scss

```diff
@@ -831,30 +831,33 @@ table.alt tfoot {
   /*font-family: FontAwesome !important;*/
 }
 
-.pubs_img {
-  width: 50%;
-  min-width: 120px;
-  display: box;
-  /* border-right: 2px solid black;
-  padding-right: 1em;
-  padding-left: 6em;
-  text-align: right; */
-  text-decoration: none !important;
-}
+// .pubs_img {
```

Comment:
This comment applies to the entire file:

Instead of commenting old code in I recommend to remove it.
If we need to revert back we can always look into the git history/blame.

I want to stop the lava-flow (dead code) antipattern from spreading/forming.